lib/typescript/render/providers/google/components/Suggestions/index.tsx

```diff
@@ -0,0 +1,118 @@
+        {text && <Text contact={contact} fromContact={fromContact} text={text} />}
+
+        {image && (
+          <Image contact={contact} fromContact={fromContact} imageUrl={image.fileUrl} altText={image.altText} />
+        )}
+
+        {!text && !image && fallback && <Text contact={contact} fromContact={fromContact} text={fallback} />}
+
+        <div className={styles.itemMemberSuggestions}>
+          <div className={styles.suggestionsContainer}>
+            {(suggestions as (
+              | SuggestedReplies
+              | SuggestedActions
+              | AuthenticationRequestSuggestion
+              | LiveAgentRequestSuggestion
+            )[]).map(elem => {
+              if ('reply' in elem) {
+                return (
+                  <button key={elem.reply.text} className={styles.replyButton}>
+                    <h1 key={elem.reply.text} className={styles.title}>
+                      {elem.reply.text}
+                    </h1>
+                  </button>
+                );
+              }
+
+              if ('action' in elem) {
+                return (
+                  <button key={elem.action.text} className={styles.replyButton}>
+                    <img
+                      className={styles.actionImage}
+                      alt={elem.action.openUrlAction ? 'link icon' : 'phone icon'}
+                      src={elem.action.openUrlAction ? linkIcon : phoneIcon}
+                    />
+                    <h1 key={elem.action.text} className={styles.title}>
+                      <a
+                        key={elem.action.text}
+                        className={styles.title}
+                        href={
+                          elem.action.openUrlAction && elem.action.openUrlAction.url
+                            ? elem.action.openUrlAction.url
+                            : `tel: ${elem.action.dialAction && elem.action.dialAction.phoneNumber}`
+                        }>
+                        {elem.action.text}
+                      </a>
+                    </h1>
+                  </button>
+                );
+              }
+
+              if ('authenticationRequest' in elem) {
+                return (
+                  <button key={elem.authenticationRequest.oauth.clientId} className={styles.replyButton}>
+                    <h1 key={elem.authenticationRequest.oauth.clientId} className={styles.title}>
+                      Sign in with Google
+                    </h1>
+                  </button>
+                );
+              }
+
+              if ('liveAgentRequest' in elem) {
+                return (
+                  <button key={Math.floor(Math.random() * 50)} className={styles.replyButton}>
+                    <h1 key={Math.floor(Math.random() * 50)} className={styles.title}>
+                      Message a live agent
+                    </h1>
+                  </button>
+                );
+              }
+            })}
+            {suggestions &&
```

Comment:
grey hover text appears on actions that can only be triggered by the contact (but I kept the external links clickable)